apps/health/boot.js

```diff
@@ -23,6 +26,35 @@
    if (Bangle.getHealthStatus().bpmConfidence) return;
    onHealth();
   } else Bangle.setHRMPower(hrm!=0, "health");
+  if (settings.withRecentMovement) {
+    let x, y, z, hadMovement = false;
+    const onAccel = (a) => {
+      const x2 = Math.round(a.x * 100),
+        y2 = Math.round(a.y * 100),
+        z2 = Math.round(a.z * 100);
+      if (x != null && (Math.abs(x - x2) > 1 || Math.abs(y - y2) > 1 || Math.abs(z - z2) > 1)) {
+        hadMovement = true;
+        Bangle.removeListener('accel', onAccel);
+      }
+      x = x2, y = y2, z = z2;
+    };
+    // look for movement over 5 seconds
+    const lookForMovement = () => {
+      hadMovement = false;
+      x = null;
+      Bangle.on('accel', onAccel);
+      setTimeout(
+        () => {
+          Bangle.removeListener('accel', onAccel);
+          paused = !hadMovement;
+          // check again in 15 or 5 minutes
+          setTimeout(lookForMovement, hadMovement ? 900000 : 300000);
```

Comment:
Perhaps `changeInterval` would be more effective here, to avoid burning through timeout ids